lib/MusicBrainz/Server/Controller/WS/2/Release.pm

```diff
@@ -195,6 +196,7 @@ sub release_toplevel {
                    uniq map { $_->artist_credit } @ac_entities;
 
         $c->model('Artist')->load(@acns);
+        $c->model('ArtistType')->load(map { $_->artist } @acns);
 
         my @artists = uniq map { $_->artist } @acns;
```

Comment:
We can move this above the added line above and call `->load(@artists)` to avoid constructing the list twice.